test/gurka/test_shortcut.cc

```diff
@@ -389,13 +359,11 @@ TEST(Shortcuts, ShortcutRestrictions) {
   baldr::GraphReader reader(map.config.get_child("mjolnir"));
 
   // test we got the right shortcuts edges, implicitly means they were broken properly
-  for (const auto& end_node : {"C", "J", "L"}) {
-    const auto shortcut =
-        gurka::findEdge(reader, layout, "highway", end_node, baldr::GraphId{}, 0, true);
+  for (const auto& pair :
+       node_pairs{{"A", "C"}, {"C", "A"}, {"H", "J"}, {"J", "H"}, {"J", "L"}, {"L", "J"}}) {
+    const auto shortcut = gurka::findEdgeByNodes(reader, layout, pair.first, pair.second);
```

Comment:
the `findEdges` is tripping now that we don't keep the name for the shortcut anymore. in the end actually _could_ keep it, it has no adverse effects other than a few more lines of redundant code (which is why I removed it)